lib/knapsack_pro/runners/test_unit_runner.rb

```diff
@@ -11,21 +11,32 @@ def self.run(args)
         if runner.test_files_to_execute_exist?
           adapter_class.verify_bind_method_called
 
-          require 'test/unit'
```

Comment:
require was hide in the `run` method instead of being at the top of the file to not load test-unit gem during loading knapsack_pro gem.

If someone uses only RSpec then he wouldn't be able to load knapsack_pro gem without having test-unit in the Gemfile. We don't want to force users to install gems they don't use in their projects.